### DISCUSSION TITLE
There are version incompatibilities between Gauge and it's plugins in this project. Some features will not work as expected.

### DISCUSSION THREAD

**Discussion by jkrn1580** [getgauge/gauge#2087] [General]
Hi, I am seeing some issue in VS code the error message shows "There are version incompatibilities between Gauge and it's plugins in this project. Some features will not work as expected." 
My gauge version details below 
Gauge version: 1.4.0
Plugins
-------
html-report (4.1.2)
js (2.3.17)
screenshot (0.1.0)

The gauge extension version in VS code is Gauge v0.0.20.

can someone please assist me with this issue . thanks.

**Comment by sriv** [comment]
what does your project's `manifest.json` look like?

**Reply by jkrn1580** [reply]
Thanks Srikanth for your replay. i am able to figure out the issue . that the js plugin was not installed properly. i had manually install using .zip file after this it worked fine
